README.md

```diff
@@ -30,7 +30,7 @@ The goal is to offer high-level APIs for developers to quickly get started in wo
 - [Scaling transforms from laptop to cluster](#laptop_cluster)
 - [Repository Use and Navigation](doc/repo.md)
 - [How to Contribute](CONTRIBUTING.md)
-- [Talks and Papers](#talks_papers)
+- [Resources (papers, talks, presentations and tutorials)](resources.md)
 - [Citations](#citations)
 
```

Comment:
Need to first sync the fork with upstream and then update the README.md to take into account the changes that were done to the upstream file.